package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@equinor/subsurface-app-management",
-  "version": "1.1.2",
+  "version": "1.1.3",
```

Comment:
need to bump to 1.1.4
1.1.3 is current release

Reply:
Pushed new commit